.github/workflows/build.yml

```diff
@@ -25,6 +29,57 @@ jobs:
       - name: Build and Test
         run: make build-and-test
 
+  run-integration-tests:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python: [ "3.10" ]
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v3
+        with:
+          python-version: "${{ matrix.python }}"
+      - name: Test Python version
+        run: |
+          installed="$(python --version)"
+          expected="${{ matrix.python }}"
+          echo $installed
+          [[ $installed =~ "Python ${expected}" ]] && echo "Configured Python" || (echo "Failed to configure Python" && exit 1)
+      - name: Install required os level applications
```

Comment:
@tzaffi There's various setup/dependency downloading happening here + subsequent `run` stanzas.  Is it all still needed now that the PR uses a pre-built sandbox image?

Reply:
I'll investigate and see which are actually still necessary.

Reply:
I whittled down the github container's bootstrapping as far as I could go. I was able to get rid of `git` and `nodejs` but needed everything else. The dependency is as follows:

`sandbox` &rarr; `docker-compose` &rarr; `docker-*` &rarr; `gpg` &rarr; `curl`

Reply:
@tzaffi Thanks for making a pass here.  I experimented further by partitioning act-only steps in https://github.com/algorand/pyteal/pull/299.

Can you have a look to see if you'd feel comfortable bringing in the change?  Otherwise, I can hold until we merge here.

Reply:
blocked by #299 discussion

Reply:
I've merged #299 with one [outstanding issue](https://github.com/algorand/pyteal/pull/299#issuecomment-1110296347)

Reply:
@tzaffi Thanks for taking in #299.  Resolving.